Products/ZenRRD/zencommand.py

```diff
@@ -492,21 +492,23 @@ def doTask(self):
 
         except Exception as e:
             self.state = TaskStates.STATE_PAUSED
-            log.exception(
-                "Task paused  name=%s device-id=%s ip=%s message=%s",
+            err_msg = "Task paused  name=%s device-id=%s ip=%s message=%s" % (
                 self.name,
                 self._devId,
                 self._manageIp,
-                e.message,
-            )
+                e.message)
+            if log.isEnabledFor(logging.DEBUG):
+                log.exception(err_msg)
+            else:
+                log.error(err_msg)
             self._eventService.sendEvent(
                 STATUS_EVENT,
                 device=self._devId,
                 summary=e.message,
                 component=COLLECTOR_NAME,
                 severity=Error,
             )
-            raise
+            defer.fail(e)
```

Comment:
This can be removed.  All `defer.fail` does is returned a `Deferred` object with its errback called, with `e` as the argument passed to the errback.